### ISSUE TITLE
Defense Skill Not raising with +defense talent

### ISSUE THREAD

**Issue by GrumpyPally** [Looking4Group/L4G_Core#3380]
35 BE Paladin - Divinia, progression realm
Anticipation talent 5/5 (+20 defense)

So I have noticed that my defense would not go above the max defense for my current level with this talent.  so if lvl 35, my defense would cap at 175/195.

Upon re-spec, my defense dropped to 155/175.  I leveled up my defense to 175 prior to taking the anticipation talent.  Upon taking the talent i had 195/195 defense.

upon leveling I have 195/200 for defense.  Despite letting 3 mobs hit me for over 30 mins, i did not receive any skill point in defense.

Is this something that can be fixed or is this talent only valid at max level?

**Comment by GrumpyPally** [comment]
3 levels have now pasted and no defense skill increase.

**Comment by Nodon** [comment]
Same for warriors protection talent, Anticipation + 20 defense, doesn't let me raise my def. skill over 350 at level 70 - it should go to 370.

**Comment by nathanielgarneau** [comment]
This appears to be working on the instant 70 server. 
Please look at the database talent records and they should match but likely are out of sync.

**Comment by nathanielgarneau** [comment]
This is not working as intended even at max level and after respeccing the talent. my characters defense is stuck at 350/370 and it is not possible to skillup beyond this

**Comment by AnonXS** [comment]
My first best guess would be that it bugs visualy for players due to increased defense rating gain set in the config.

**Comment by nathanielgarneau** [comment]
I will check and try to confirm via ingame api calls if the returned values match the expected.

**Comment by nathanielgarneau** [comment]
this is showing as incorrect and even the values returned for dodge/parry/block based on defense imply that the talented defense stats are not being counted. 
check defense rating from gear 
/script DEFAULT_CHAT_FRAME:AddMessage(GetCombatRating(CR_DEFENSE_SKILL))
check defense skill from player , this should report the talents anticipation +20, on my paladin menteith, it reports a value of 351 which is an impossible value to attain. perhaps this is a data issue. 
/script DEFAULT_CHAT_FRAME:AddMessage(UnitDefense("player"))
this is the conversion from defense rating to defense skill. and this does appear valid. 
/script DEFAULT_CHAT_FRAME:AddMessage((GetCombatRating(CR_DEFENSE_SKILL)*150/355))

**Comment by nathanielgarneau** [comment]
That is true. But also the defense talents did not up the skill page from 350/350 to 370/370 in retail. This skill page value is possible to check against , cr_defense_skill . as implemented this +20 is already included and need not be hard coded. 

Either way there are data issues for players who see 350/370 or 351/370 .
